classes/classes/Items/ConsumableLib.as

```diff
@@ -210,7 +210,7 @@ package classes.Items
 		public const VIXVIGR:SimpleConsumable = mk("VixVigr","VixVigr", "a bottle labelled \"Vixen's Vigor\"", curry(m.foxTF, true), "This small medicine bottle contains something called \"Vixen's Vigor\", supposedly distilled from common fox-berries.  It is supposed to be a great deal more potent, and a small warning label warns of \"extra boobs\", whatever that means.", 30);
 		public const W_FRUIT:SimpleConsumable = mk("W.Fruit","W.Fruit", "a piece of whisker-fruit", m.whiskerFruit, "This small, peach-sized fruit has tiny whisker-like protrusions growing from the sides.");
 		public const WETCLTH:SimpleConsumable = mk("WetClth","WetClth", "a wet cloth dripping with slippery slime", m.wetCloth, "Dripping with a viscous slime, you've no doubt rubbing this cloth on your body would have some kind of strange effect.");
-		public const WOLF_PP:SimpleConsumable = mk("Wolf Pp","WolfPepper", "a Wolf Pepper", m.wolfPepper, "The pepper is shiny and black, bulbous at the base but long and narrow at the tip. It has a fuzzy feel to it and it smells spicy. Somehow, you know it's different from the usual Canine Peppers you see.");
+		public const WOLF_PP:SimpleConsumable = mk("Wolf Pp","WolfPepper", "a Wolf Pepper", curry(m.wolfPepper, 0), "The pepper is shiny and black, bulbous at the base but long and narrow at the tip. It has a fuzzy feel to it and it smells spicy. Somehow, you know it's different from the usual Canine Peppers you see.");
```

Comment:
It's not meant to have a curry. The type=0 was a mess-up on my part-- I've removed it in my PR. Forgot to delete it when I was working off of Canine Peppers. Would advise removing this before Kitteh merges, we don't want the code to get fuck-y